src/include/s390/onpanic/ui.rb

```diff
@@ -312,6 +315,23 @@ def OnPanicDialog
       ret
     end
 
+    # Check if kdump is enabled. If yes, ask the user to disable it
+    # because it conflicts with OnPanic
+    def check_kdump
+      if OnPanic.start &&
+          (Service.Enabled(KDUMP_SERVICE_NAME) || Service.Active(KDUMP_SERVICE_NAME))
+        if Yast::Popup.YesNo(
+          _(
+            "The service kdump is active and will conflict with dumpconf.\n" \
```

Comment:
As we already have the `KDUMP_SERVICE_NAME` maybe we could also use it in the message. That will avoid changing (and breaking) the translation if the name is changed in the future.